### DISCUSSION TITLE
Minimalist-setup with no/fewer icons

### DISCUSSION THREAD

**Discussion by Aceticia** [LunarVim/LunarVim#2411] [Q&A] [answered]
I've been loving the snap and ease that lunarvim brings. My own configs are always somewhat janky, and lunarvim is the only thing preventing me from going back into the mousy hell of VSCode. Functionally, I wouldn't change a thing about LunarVim.

But I do wonder if I can make it look a bit more minimalist like my previous setups. More specifically, I wonder if it's possible to turn off parts of the icons. This will be desirable for two reasons:
1. On some servers it might be difficult to support them and thus it end up looking really weird
2. They are less distracting in daily situations

How would you go about doing this? The (somewhat) unnecessary symbols (for me) are those in sign column, those in the explorer, those in the buffer line, also those in the status line. (Probably all of them)

**Comment by c02y** [comment]
+1

I don't use any nerd-font, so I want to disable any icon in vim as well.

**Comment by abzcoding** [comment, marked as answer]
fixed via #2529
